sdk/storage/storage-blob/test/perfstress/track-2/upload.spec.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { StorageBlobTest } from "./storageTest.spec";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+dotenv.config();
+
+interface StorageBlobUploadTestOptions {
+  size: number;
+}
+
+export class StorageBlobUploadTest extends StorageBlobTest<StorageBlobUploadTestOptions> {
+  public options: PerfStressOptionDictionary<StorageBlobUploadTestOptions> = {
+    size: {
+      required: true,
+      description: "Size in bytes",
+      shortName: "sz",
+      longName: "size",
+      defaultValue: 10
+    }
+  };
+
+  async runAsync(): Promise<void> {
```

Comment:
I recommend uploading to the same blob per instance, and also reusing the upload buffer.

Reply:
Done!